test/features/linkchecker.feature

```diff
@@ -0,0 +1,24 @@
+@api @enableDKAN_Linkchecker @disablecaptcha
+Feature:
+  Linkchecker tests for DKAN Linkchecker Module
+
+  Background:
+    Given users:
+      | name    | mail                | roles                |
+      | John    | john@example.com    | site manager         |
+
+    Given groups:
```

Comment:
We are not using the groups or group memberships on the test, I think? Why are we creating them?

Reply:
This is so the test passes on client sites that require group assignment on their datasets

Reply:
ok after looking again I remember I removed the test that created a dataset with a bad link because too much needs to happen to get that bad link into the report. So since we are not creating a dataset I've removed the group background setup.